code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -890,6 +890,9 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/monkeyburger/New()
 	..()
+	if(rand(1))
```

Comment:
this is true 100% of the time

rand(1) returns a number from 1 to 1, 1 = TRUE

you want prob (1)

this is funny because I made a joke like this in the lootbox PR thread

Reply:
wow you fucking RUINED IT

Reply:
oh you know, I actually investigated this and it's only 50% of the time, if only one arg is given it's treated as the upper bound

still I think you meant prob

Reply:
🍆

Reply:
@Kurfursten no I didn't and please delete your comment so nobody else notices